starwars/src/components/CharacterList.js

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+
+import Character from './Character';
+
+export default function CharacterList(props) {
+  return (
+    <div className="character-list">
+      {props.starwarsChars.map(character => {
+        return (
+          <Character
+          character={character}
+          gender={character.gender}
+          height={character.height}
+           />
```

Comment:
a common react pattern would be to separate these concerns - one component would do the mapping work, and another would be responsible for the render of the character itself